### PR TITLE
LPS-152623 Increase the segments experience priority, so when we create the default segment experience it can have the priority 0

### PR DESCRIPTION
**Pull request by ealonso**
Motivation
===

Ensure that the default SegmentsExperience priority is 0.

https://issues.liferay.com/browse/LPS-152623

Proposed Solution
===

Increases the segments experiences priorities when they are greater than 0. In that way, we can have the 0 priority for default experiences. 

Steps to Verify:
---
1. Run a 7.2
2. Creates a content page
3. Adds a new segments experience
4. Upgrade to master

### CODE REVIEW

**Comment by ealonso** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by brianchandotcom** [comment]
@ealonso merged it manually

File: modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/upgrade/v2_6_0/SegmentsExperienceUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+				"update SegmentsExperience set priority = ? where " +
+					"segmentsExperienceId = ?")) {
+
+			preparedStatement.setInt(1, priority + 1);
+			preparedStatement.setLong(2, segmentsExperienceId);
+
+			preparedStatement.executeUpdate();
+		}
+		catch (Exception exception) {
+			if (_log.isDebugEnabled()) {
```
Comment (cgoncas) [inline modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/upgrade/v2_6_0/SegmentsExperienceUpgradeProcess.java:48]: Do you want this error is shown only in debug mode? This is a new rule for upgrade processes?

**Comment by cgoncas** [comment]
@ealonso !!

I have closed it, because Brian already merged it, I have the doubt about he logs, but as a Source Formatter doubt.


Thanks!!!

File: modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/upgrade/v2_6_0/SegmentsExperienceUpgradeProcess.java (hunk 1 above)
Comment (ealonso) [reply modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/upgrade/v2_6_0/SegmentsExperienceUpgradeProcess.java:48]: Hey @cgoncas,

I just copied the Upgrade process from com.liferay.segments.internal.upgrade.v2_0_0.SegmentsExperienceUpgradeProcess

If you want, we can change both logs and replace for _log.error

Comment (ealonso) [reply modules/apps/segments/segments-service/src/main/java/com/liferay/segments/internal/upgrade/v2_6_0/SegmentsExperienceUpgradeProcess.java:48]: Please let me know and I send you a follow up pull

**Pull request closed without merging**